settings.js

```diff
@@ -157,18 +157,26 @@ function _migrateToV1(migrationSettings, extensionSettings) {
                 Shell.util_get_translated_folder_name(itemId);
 
             id = _createFolder(folderSettings, translatedName, folderIcons);
-        } else {
-            if (!installedAppsSet.has(itemId))
-                continue;
+        } else if (!installedAppsSet.has(itemId)) {
+            continue;
+        }
+
+        // If we have more than 24 icons, make sure that the app center icon
+        // appended to the first page
+        if (index === 23) {
```

Comment:
Can we use `itemsPerPage - 1` here instead? I think it is better than hardcoding the 23 (or at least put it in an const/enum I guess).